package.json

```diff
@@ -75,7 +75,6 @@
     "@types/node": "^18.18.5",
     "@types/react": "^16.14.20",
     "@types/react-dom": "^16.9.14",
-    "@types/react-resizable": "^1.7.4",
```

Comment:
The actual dependency was removed long ago, but the types stayed: https://github.com/cloudscape-design/components/pull/616